gbpservice/nfp/configurator/drivers/base/base_driver.py

```diff
@@ -0,0 +1,49 @@
+import subprocess
+from oslo_log import log as logging
+LOG = logging.getLogger(__name__)
+SUCCESS = 'SUCCESS'
+FAILED = 'FAILED'
+
+"""Every service vendor must inherit this class. If any service vendor wants
+   to add extra methods for their service, apart from below given, they should
+   add method definition here and implement the method in their driver
+"""
+
+
+class BaseDriver(object):
+    def __init__(self):
+        pass
+
+    def configure_interfaces(self, context, kwargs):
```

Comment:
These are not the API that is exposed and are listed in blue print. They are internal methods of configurator/service agents